client/src/components/ElementEditor/ElementActions.js

```diff
@@ -59,12 +60,15 @@ class ElementActions extends Component {
    * @returns {DropdownItem|null}
    */
   renderDivider() {
-    const { children, editTabs } = this.props;
+    const { children, editTabs, expandable } = this.props;
 
-    if (editTabs && editTabs.length && React.Children.count(children)) {
```

Comment:
I swap the return null and return DropdownItem around just so the approach between renderDivider and renderTabs are consistien